lib/facility_factory.rb

```diff
@@ -0,0 +1,54 @@
+class FacilityFactory
+  attr_reader :or_facilities,
+              :ny_facilities,
+              :mo_facilities
+  
+  def initialize
+    @or_facilities = []
+    @ny_facilities = []
+    @mo_facilities = []
+  end
+
+  def create_or_facilities(dmv_data)
+    dmv_data.map do |facility|
```

Comment:
Overall nice job here. 
For your enumerable choice, I would have just done #each since you are not caring about the return value of the enumerable action, rather, you are putting all the new objects into the facility arrays so no map is needed.